integrations/haystack/README.md

```diff
@@ -0,0 +1,305 @@
+
+## Accuracy Evaluation ##
+The DeepSparse nodes were evaluated using evaluation scripts provided by Tevatron. These results are consistent with those documented in [Dense Passage Retrieval for Open-Domain Question Answering](https://arxiv.org/abs/2004.04906).
+
+|Model (WikipediaNQ)|Accuracy @20|Recall|Accuracy @100|Recall|Accuracy @ 200|Recall|
+|-|-|-|-|-|-|-|
+|base-none|0.7983379501|100.00%|0.863434903|100.00%|0.8842105263|100.00%|
+|base-none-untied|0.7988919668|100.07%|0.8581717452|99.39%|0.8842105263|100.00%|
+|pruned90-none|0.7878116343|98.68%|0.8584487535|99.42%|0.8770083102|99.19%|
+|pruned90-none-untied|0.7828254848|98.06%|0.8570637119|99.26%|0.8747922438|98.93%|
+pruned80-vinni|0.7847645429|98.30%|0.856232687|99.17%|0.8717451524|98.59%|
+pruned80-vinni-untied|0.7817174515|97.92%|0.8509695291|98.56%|0.8717|98.59%|
+
+|Model (TriviaNQ) |Accuracy @20|Recall|Accuracy @100|Recall|Accuracy @ 200|Recall|
+|-|-|-|-|-|-|-|
+|base-none|0.7961637055|100.00%|0.853266154|100.00%|0.8672323875|100.00%|
+|base-none-untied|0.7943074339|99.77%|0.8503491558|99.66%|0.8661716609|99.88%|
+|pruned90-none|0.7839653496|98.47%|0.8440731901|98.92%|0.8609564218|99.28%|
+|pruned90-none-untied|0.782904623|98.33%|0.8435428268|98.86%|0.8594537258|99.10%|
+|pruned80-vinni|0.7930699196|99.61%|0.8480509149|99.39%|0.8649341466|99.73%|
+|pruned80-vinni-untied|0.7867939539|98.82%|0.8460178556|99.15%|0.8629010872|99.50%|
+
+|MSMARCO Passage|MRR@10|Recall|Accuracy @10|Recall|Recall@20|Recall|Recall@100|Recall|Recall@200|Recall|
+|-|-|-|-|-|-|-|-|-|-|-|
+|base-none|0.3220429117|100.00%|0.6021489971|100.00%|0.6979942693|100.00%|0.8528653295|100.00%|0.8951289398|100.00%|
+|base-none-untied|0.3209568722|99.66%|0.5984240688|99.38%|0.6892550143|98.75%|0.8484240688|99.48%|0.8928366762|99.74%|
+|pruned90-none|0.3276589007|101.74%|0.6146131805|102.07%|0.7004297994|100.35%|0.8537249284|100.10%|0.8932664756|99.79%|
+|pruned90-none-untied|0.3093550166|96.06%|0.588252149|97.69%|0.6793696275|97.33%|0.8368194842|98.12%|0.8812320917|98.45%|
+|pruned80-vinni|0.3251235958|100.96%|0.6068767908|100.79%|0.6962750716|99.75%|0.8449856734|99.08%|0.8856733524|98.94%|
+|pruned80-vinni-untied|0.3124041479|97.01%|0.5918338109|98.29%|0.6802292264|97.45%|0.8319484241|97.55%|0.8780802292|98.10%|
+
+## Performance Evaluation ##
+Retrievers were also evaluated on their run time. This table compares the run time of generating query embeddings using `DenseEmbeddingRetriever` with Pytorch and `DeepSparseEmbeddingRetriever` with the DeepSparse Engine. Both retrievers were evaluated with the same 12 layer BERT model on the same hardware using CPU.
+
+|Number of Queries|DenseEmbeddingRetriever (sec)|DeepSparseEmbeddingRetriever (sec)|
+|-|-|-|
+|1|0.027|0.012|
+|100|1.37|0.96|
+|1,000|12.6|9.5|
```

Comment:
nit: would be best to run this with a quantized model

Reply:
Will add this in a followup PR